core/src/debug/assets/mobile-core.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Actually best to remove all assets and test it in top level example project (we will need to have it here to test if this is usable`. WDYT?

From android docs:
`Library projects cannot include assets`

Reply:
Testing in a top level project has the problem that that project will have to be run in an emulator which we are trying to avoid.